libs/etterlatte-brev-model/src/main/kotlin/no/nav/etterlatte/brev/Brevkoder.kt

```diff
@@ -115,9 +115,9 @@ enum class Brevkoder(
         "Vedtak om endring av omstillingsstønad",
         Brevtype.VEDTAK,
     ),
-    OMS_INNTEKTSJUSTERING(
```

Comment:
Er det oppretta brev av denne typen i dev/prod tidlegare? I så fall vil det fort feile, sia vi lagrar Brevkoder i brev-tabellen. Så sjekk det først.

Reply:
Skal ikke være oppretta brev ettersom jobben for inntektsjustering aldri har blitt kjørt, men skal ta en ekstra sjekk!